### ISSUE TITLE
Show disused and abandoned and razed tracks in different colours for accessibility

### ISSUE THREAD

**Issue by erebion** [OpenRailwayMap/OpenRailwayMap-CartoCSS#116]
Hello,

please change the colours for tracks that are disused to something else that is not the same as abandoned or razed.

It's pretty hard to spot the difference. This is also an accessibility issue. Ideally users could choose what to display without relying on slight colour differences at all.

I'm trying to learn more about disused tracks im my area and it often happens that I think I found one, zoom in and then it is actually abandoned or razed. Also, the difference between abandoned and razed is even harder to see.

I'd also like some clarification as to what the differences between disused and abandoned are. To me it sounds like the same. Razed seems like "non-existent". Is that a correct interpretation?

**Comment by Nakaner** [comment]
Disused and abandoned tracks have a similar colour but you can also distinguish them by the dashing. Abandoned tracks are dashed (5,5), disused tracks are not dashed. However, dashing does not look as expected on lower zoom levels because individual OSM ways are rendered as individual linestrings and therefore, dashing restarts at each node where the ways are split up (see #1).

**Comment by erebion** [comment]
I know that abandoned tracks are dashed. However, that is really hard to see.

Maybe we could let users decide the colour or use a different colour.

Ideally users could choose what tracks do display, so even people that do not have colour perception can use the map by only enabling what they are interested in.

**Comment by besentv** [comment]
I have an idea, which i still need to test though. How about using a thin non dotted line with the razed tracked color at low zoom and at high zoom values make the line for razed dotted? I think at high zoom it isn't fascinating anyway if a track is just demolished or built over?

**Comment by erebion** [comment]
Nah, I'm asking specifically because I want to find those tracks, so I have to be able to see them at low zoom levels. :D

**Comment by besentv** [comment]
So my first idea was to do something like this, where disused, abandoned and razed tracks only use colors brown - black. Black: disused, brown: abandoned, razed: dashed
Yard is now blueish purple and industrial tracks are pink-ish.

![image](https://github.com/OpenRailwayMap/OpenRailwayMap-CartoCSS/assets/10588728/a718afb5-efc6-4ba4-9655-7e5aa5afa4cb)
![image](https://github.com/OpenRailwayMap/OpenRailwayMap-CartoCSS/assets/10588728/eb7f0789-a0a0-4816-9cd8-b14886028dab)
